giturlparse/parser.py

```diff
@@ -70,29 +70,32 @@ def parse(self):
             'owner': None,
         }
 
-        regexp = (r'^(https?|git|ssh|rsync)\://'
+        regexp = []
+        regexp1 = (r'^(https?|git|ssh|rsync)\://'
                   '(?:(.+)@)*'
                   '([a-z0-9_.-]*)'
                   '[:/]*'
                   '([\d]+){0,1}'
                   '(/(.+)/(.+).git)')
-        m1 = re.search(regexp, self._url)
+        regexp.append(regexp1)
 
-        regexp = (r'^(?:(.+)@)*'
+        regexp2 = (r'^(?:(.+)@)*'
                   '([a-z0-9_.-]*)[:/]*'
                   '([\d]+){0,1}'
                   '([:/](.+)/(.+).git)')
-        m2 = re.search(regexp, self._url)
-
-        if m1:
+        regexp.append(regexp2)
```

Comment:
I's prefer we not have the temporary regexp1 and regexp2 variables and simply assign the regexp directly into the list on regexp definition.